dftimewolf/lib/module.py

```diff
@@ -64,7 +64,7 @@ def SetupLogging(self, threaded: bool = False) -> None:
         threaded=threaded))
     self.logger.addHandler(file_handler)
 
-    if not self.state.cdm:
+    if not isinstance(self.state, state_lib.DFTimewolfStateWithCDM):
```

Comment:
Can we have a boolean / property in `state` that indicates whether or not it's a CDM state? So we can have 

```python
if not self.state.iscursed:
```

or something.

Same goes for other places where `isinstance` is called

Reply:
Going the subclass method, I didn't want to have anything referring to curses in the base class. But, we can for sure have a bool `stdout_log` which accomplished the same thing. 

As for the other references, I've refactored `PublishMessage` to acheive the same goal of keeping curses pieces away from state base class and `module`.